### DISCUSSION TITLE
使用vite的vue-tsc打包element-plus的rate时报错 error TS1005: ',' expected.

### DISCUSSION THREAD

**Discussion by GamesSoul** [element-plus/element-plus#7030] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/37973776/162114524-84836b11-e9a2-4c24-b4fc-93aeda854b26.png)
目测应该是类型引入的写法问题 
我在node_modules里的rate.vue.d.ts把引入类型方式改为import type { CSSProperties }  from 'vue 就没报错了
麻烦帮忙看一下

**Comment by sxzz** [comment, marked as answer]
尝试升级 TypeScript 版本

**Reply by GamesSoul** [reply]
> 尝试升级 TypeScript 版本

还是一样的报错   typescrpit已经是用的npm库里的[4.6.3](https://www.npmjs.com/package/typescript/v/4.6.3)最新的了  还是一样有问题

**Reply by matiboy** [reply]
Had the same issue; updating to typescript 4.6.4 and vue-tsc 0.34.11 worked

**Comment by sxzz** [comment]
Please provide a minimum reproduction repo.
